CMakeLists.txt

```diff
@@ -50,7 +54,10 @@ endif()
 
 if (WIN32)
     install(TARGETS ${PROJECT_NAME}
-        RUNTIME DESTINATION lib)
+        ARCHIVE DESTINATION lib
+        RUNTIME DESTINATION bin
+        PUBLIC_HEADER DESTINATION include/kaitai
+    )
 else()
```

Comment:
Does this branching to Windows and non-Windows systems actually serve any purpose? I mean, why not just

```diff
-if (WIN32)
     install(TARGETS ${PROJECT_NAME}
         ARCHIVE DESTINATION lib
         RUNTIME DESTINATION bin
+        LIBRARY DESTINATION lib
         PUBLIC_HEADER DESTINATION include/kaitai
     )
-else()
-    install (TARGETS ${PROJECT_NAME}
-        LIBRARY DESTINATION lib
-        PUBLIC_HEADER DESTINATION include/kaitai
-    )
-endif()
```

...? The result will be the same, right?

Reply:
Yes, the result is the same. I preferred a more explicit way in order to make cross-platform support maintenance simpler. Do you want me to change that to your snippet? I don't mind it, just let me know.

Reply:
Am I correct that ARCHIVE, RUNTIME and LIBRARY are simply set to their default values? If so, you might want to discuss removing them in favour of making explicit that PUBLIC_HEADER is the only non-default setting changed for a reason. Though, explicitly documenting the used paths might have a benefit as well of course.

https://cmake.org/cmake/help/latest/command/install.html#installing-targets